src/proj_internal.h

```diff
@@ -899,6 +899,11 @@ void   pj_freeup_plain (PJ *P);
 
 PJ* pj_init_ctx_with_allow_init_epsg( projCtx_t *ctx, int argc, char **argv, int allow_init_epsg );
 
+std::string PROJ_DLL pj_add_type_crs_if_needed(const std::string& str);
+
+PJ *proj_create_internal (PJ_CONTEXT *ctx, const char *definition);
```

Comment:
Per https://github.com/OSGeo/proj.4/issues/678, I suggest renaming to `pj_create_internal` and `pj_create_argv_internal`.

Reply:
why _argv ? Did you mean pj_create_internal ?

Reply:
Adressed by renaming proj_create_internal() to pj_create_internal() and proj_create_argv_internal() to pj_create_argv_internal()